tests/msc3083_test.go

```diff
@@ -159,3 +162,159 @@ func TestRestrictedRoomsRemoteJoin(t *testing.T) {
 	// Execute the checks.
 	CheckRestrictedRoom(t, alice, bob, space, room)
 }
+
+// Request the room summary and ensure the expected rooms are in the response.
+func RequestAndAssertSummary(t *testing.T, user *client.CSAPI, space string, expected_rooms []interface{}) {
+	// The room appears for no one at first since hs2 doesn't know about who is in ss1.
```

Comment:
This comment lies, move it to where you call `RequestAndAssertSummary`

Reply:
Good catch! Thanks. :)